src/target/riscv/riscv-013.c

```diff
@@ -2843,18 +2900,24 @@ static int sample_memory_bus_v1(struct target *target,
 							(!sbaddress1_valid ||
 							sbaddress1 != config->bucket[i].address >> 32)) {
 						sbaddress1 = config->bucket[i].address >> 32;
-						riscv_batch_add_dm_write(batch, DM_SBADDRESS1, sbaddress1, true);
+						riscv_batch_add_dm_write(batch, DM_SBADDRESS1,
+								sbaddress1, true,
+								RISCV_DELAY_SYSBUS_READ);
```

Comment:
Similarly to previous comment - writing to SBADDRESS1 does not start the bus operation yet.

```suggestion
						riscv_batch_add_dm_write(batch, DM_SBADDRESS1,
								sbaddress1, true,
								RISCV_DELAY_BASE);
```

Reply:
Addressed.